doc/ale.txt

```diff
@@ -398,6 +400,7 @@ Notes:
 * AsciiDoc: `alex`!!, `proselint`, `redpen`, `write-good`
 * Awk: `gawk`
 * Bash: `language-server`, `shell` (-n flag), `shellcheck`, `shfmt`
+* BibTex: `bibclean`
```

Comment:
super nitpick, the likes of which have never seen before:

shouldn't this be BibTeX ?

Reply:
Yes, it should.  Fixed.